core/templates/services/keyboard-shortcut.service.spec.ts

```diff
@@ -33,13 +32,12 @@ describe('Keyboard Shortcuts', () => {
   beforeAll(() => {
     skipButton.setAttribute('id', 'skipToMainContentId');
     backButton.setAttribute('id', 'backButtonId');
-    nextButton.setAttribute('class', 'protractor-test-next-button');
-    continueButton.setAttribute('class', 'protractor-test-continue-button');
-    continueToNextCardButton.setAttribute(
-      'class', 'protractor-test-continue-to-next-card-button');
+    nextButton.setAttribute('class', 'oppia-next-button');
+    continueButton.setAttribute('class', 'oppia-learner-confirm-button');
     searchBar.setAttribute(
-      'class', 'protractor-test-search-input');
-    categoryBar.setAttribute('id', 'categoryBar');
+      'class', 'oppia-search-bar-text-input');
+    categoryBar.setAttribute(
+      'class', 'oppia-category-bar-dropdown-toggle');
```

Comment:
Can we move this to the line above? (Check this in other places as well)

Reply:
Um sorry what lines are you talking about? 38-40?